src/libcore/num/mod.rs

```diff
@@ -1147,6 +1295,13 @@ macro_rules! uint_impl {
         /// negative minimal value for the type); this is a positive
         /// value that is too large to represent in the type. In such
         /// a case, this function returns `MIN` itself.
+        ///
+        /// # Examples
+        ///
+        /// ```rust
+        /// assert_eq!(100i8.wrapping_neg(), -100);
```

Comment:
I think that's the old travis build; I pushed a second commit shortly after, which [built successfully](https://travis-ci.org/rust-lang/rust/builds/90069616). Will follow up on the other comments